launchers/junit/src/testFixtures/java/org/eclipse/dataspaceconnector/junit/launcher/EdcExtension.java

```diff
@@ -25,13 +25,18 @@
 import org.eclipse.dataspaceconnector.system.ExtensionLoader;
 import org.eclipse.dataspaceconnector.system.ServiceLocator;
 import org.eclipse.dataspaceconnector.system.ServiceLocatorImpl;
-import org.junit.jupiter.api.extension.*;
+import org.junit.jupiter.api.extension.AfterTestExecutionCallback;
+import org.junit.jupiter.api.extension.BeforeTestExecutionCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolutionException;
+import org.junit.jupiter.api.extension.ParameterResolver;
 
 import java.util.ArrayList;
 import java.util.LinkedHashMap;
 import java.util.List;
 
-import static org.eclipse.dataspaceconnector.common.Cast.cast;
+import static org.eclipse.dataspaceconnector.common.types.Cast.cast;
 
 /**
  * A JUnit extension for running an embedded DA-GX runtime as part of a test fixture.
```

Comment:
rename DA-GX

Reply:
Hi Moritz, let's do this change in a separate PR to catch all of those.